aspnetcore/migration/proper-to-2x/index.md

```diff
@@ -61,11 +64,11 @@ ASP.NET Core uses a similar approach, but doesn't rely on OWIN to handle the ent
 
 `Startup` must include a `Configure` method. In `Configure`, add the necessary middleware to the pipeline. In the following example (from the default web site template), several extension methods are used to configure the pipeline with support for:
 
-* [BrowserLink](http://vswebessentials.com/features/browserlink)
-* Error pages
-* Static files
-* ASP.NET Core MVC
-* Identity
+- [BrowserLink](http://vswebessentials.com/features/browserlink)
```

Comment:
To avoid warnings from Markdownlint, let's use asterisks instead of dashes in unordered lists.